languageServer/src/LanguageServer.ts

```diff
@@ -10,16 +10,14 @@ import {
   Location,
   Position,
```

Comment:
What should be shown if a file hasn't been opened yet? Currently, it appears to take us somewhere that doesn't seem to make sense: 

https://user-images.githubusercontent.com/43496356/194176758-b776d5b4-f84e-4d95-a8a1-f1156b5aee34.mov